cmd/libsnap-confine-private/feature.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef SNAP_CONFINE_FEATURE_H
+#define SNAP_CONFINE_FEATURE_H
+
+#include <stdbool.h>
+
+typedef enum sc_feature_flag {
+	SC_PER_USER_MOUNT_NAMESPACE,
```

Comment:
If you change the PR maybe SC_FEATURE_PER_USER_MOUNT_NAMESPACE (i.e. prefix with SC_FEATURE_) but not really important.

Reply:
I like the SC_FEATURE idea. I'll change this in the next one.